src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    private static int totalTasks = 0;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        totalTasks ++;
+    }
+    //"\u2713" = tick symbol, "\u2718" = cross symbol

```

Comment:
Good of you to comment this so that readers can know what the unicode symbols are